### ISSUE TITLE
Edit panel opens in PDP even when edit mode is not enabled

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2989]
### Expected behavior
Clicking on a variant should not open the edit panel when edit mode is not enabled

### Actual Behavior
Clicking on a variant to select it opens the edit panel

### Steps to Reproduce the Behavior
1. Login as admin
1. Turn off edit mode
1. Select a variant
1. Observe that the edit panel opens
![edit_mode](https://user-images.githubusercontent.com/72819/31059600-7ab4996a-a737-11e7-86dc-f345d42a5846.gif)

### Versions
```
Node: 7.10.0
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.13.0
Reaction: 1.5.0
Reaction branch: marketplace
Docker: 17.06.2-ce
```

**Comment by aanchirinah** [comment]
I just replicated the bug and I'm on it

**Comment by spencern** [comment]
Resolved by #3039
